### DISCUSSION TITLE
Documentation: how do you write and build it?

### DISCUSSION THREAD

**Discussion by pawamoy** [explosion/spaCy#6931] [Help: Other Questions] [answered]
Hi, I think your docs are gorgeous, and I was wondering how you write them, and how you build them?
I see you have Markdown tables for methods parameters and such. Are they auto-generated or written by hand?
What static site generator are you using? I couldn't find it with a quick glance to the source code, sorry :sweat_smile:

**Comment by pawamoy** [comment, marked as answer]
Ah I see, it's Gatsby :smile:

**Reply by svlandeg** [reply]
Yep, the readme [here](https://github.com/explosion/spaCy/tree/master/website) has some more info :-)

**Reply by pawamoy** [reply]
Perfect, thanks! I can't mark your reply as the answer so I'm marking mine instead :sweat_smile:
